src/auth.test.js

```diff
@@ -30,7 +30,8 @@ describe('auth', () => {
       unlink: jest.fn(),
       updateEmail: jest.fn(),
       updatePassword: jest.fn(),
-      updateProfile: jest.fn()
+      updateProfile: jest.fn(),
+      deleteProfile: jest.fn()
```

Comment:
I think your problem is here: [the method is called `delete` and not `deleteProfile`](https://firebase.google.com/docs/reference/js/firebase.User#delete). ;)

Reply:
Oops, thanks